benchmarks/dynamo/common.py

```diff
@@ -1101,18 +1127,26 @@ def load(cls, model, example_inputs, eager_forward):
             example_outputs = eager_forward(
                 copy.deepcopy(model), clone_inputs(example_inputs)
             )
+
             if isinstance(example_outputs, dict):
                 # Workaround Huggingface output type issue ModelOutput
                 example_outputs = dict(example_outputs)
             example_outputs, output_spec = pytree.tree_flatten(example_outputs)
             for output in example_outputs:
                 output_tensors.append(torch.empty_like(output))
 
-            # The exact API is subject to change
-            exported = torch._export.export(model, example_inputs)
+            # Register the output dataclass to pytree
+            example_outputs = eager_forward(
```

Comment:
Why are we calling eager_forward again? (we should really get rid of the first one too)

Reply:
We need to register the dataclasses for the outputs, otherwise dynamo fails when checking that the outputs are equal :/

Reply:
I got rid of the first call to eager_forward, but I'm not sure how to get rid of the second call (to get the output dataclass types)